style.css

```diff
@@ -1 +1,126 @@
-/*  Add your styling here */
+.grid {
+    /* border: solid 1px red; */
+
+}
+
+.grid-one {
+    display: grid;
+    grid-template-columns: 140px 140px;
+    /* border: solid 1px pink; */
+}
+
+.grid-one>img {
+    margin: auto;
+}
+
+.grid-one>h1 {
+    margin: auto;
+}
+
+@media screen and (min-width:900px) {
+    .grid-one {
+        display: flex;
+    }
+
+    .grid-one>img {
+        margin-left: 20px;
+    }
+
+    .grid-one>h1 {
+        margin-right: 100px;
+    }
+
+    .grid-two {
+        display: flex;
+        justify-content: space-around;
+    }
+
+    .general-hero-section {
+        display: flex;
+        padding: 10px;
```

Comment:
You might want all your sections to line up on the edges of the screen. This section seems a little more indented on desktop than the others, and your `pictures-section` also has a different spacing. Try and stay consistent for your design to look nice and neat 🙂